webcrypto/tests/util/testharness.js

```diff
@@ -91,13 +91,3 @@ function assert_in_array(actual, expected, description) {
 function assert_unreached(description) {
     throw `reached unreachable code, reason: ${description}`
 }
```

Comment:
Do we need this "adaptation" ? can we have a diff for it as well?

Reply:
Good question, it's something from the original implementation which probably should [cover this](https://github.com/web-platform-tests/wpt/blob/master/resources/testharness.js#L2375-L2379), let me check quickly if it can be dropped

Reply:
Well, quickly checking, I think we need this :cry: It's the only place where it's defining, keeping in mind that we don't fully rely on the original `testharness.js` and `helpers.js`

Reply:
Yeah, but the questions is more "can we just really on the original ones?" 

Or can we have diff patch against the original ones (like we have for the tests) and run them patched instead.

Reply:
Probably, yes. Not sure, though, if it's just a simple replacement. Perhaps if it was possible, we started to use them at the moment of beginning the webcrypto. How about opening a separate issue? Ideally, we should use the original test framework as much as possible.

Reply:
Okay, after re-looking to streams, it seems like it's not a big deal, so I applied the fixed. It removes the need to any overload (even if test time increased a bit :thinking: )